### ISSUE TITLE
Snap.version returns incorrect value for v0.4.1

### ISSUE THREAD

**Issue by h4** [adobe-webplatform/Snap.svg#372]
I've installed Snap.svg == 0.4.1 via Bower and type `Snap` inside console. Suddenly i've got "0.4.0".

On last commit (84fbff7d512c8145c522b71fc9c872cb0bcae49a) you bumped up library version, but Bower uses git tags, and tag 0.4.1 linked with older commit de1f20748bcf81a8bdd213a24f3f5c471ec706ce.

**Comment by Siyfion** [comment]
I agree with @h4, this is going to cause issues with a lot of build-systems including, among others, the Meteor Packaging project. Can we remove the "v0.4.1" tag and re-add it to the correct commit version @DmitryBaranovskiy?
